app/main/db/queries.py

```diff
@@ -81,15 +83,53 @@ def build_fuzzy_search_query(query_string: str, sorting_orders=None):
         ).label("opening_date"),
     )
 
+    if transferring_body_id:
+        query = query.filter(
+            sub_query.c.transferring_body_id == transferring_body_id
+        )
+
     if sorting_orders:
         query = _build_sorting_orders(query, sub_query, sorting_orders)
-    else:
-        query = query.order_by(
-            sub_query.c.transferring_body,
-            sub_query.c.series,
-            sub_query.c.consignment_reference,
-            sub_query.c.file_name,
+    # else:
```

Comment:
can this be removed?

Reply:
I removed it in the new ticket AYR-765 which I am currently working on if that is ok